src/Balancer/Group.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Utopia\Balancer;
+
+class Group
+{
+    /**
+     * @var Balancer[]
+     */
+    protected array $balancers = [];
+
+    /**
+     * @param Balancer $balancer
+     * @return self
+     */
+    public function add(Balancer $balancer): self
+    {
+        $this->balancers[] = $balancer;
+        return $this;
+    }
+
```

Comment:
Didn't we want to add a  `setStrategy` functions and have some constant strings for the strategy types?

Reply:
Yes, but we would never use it. I considered it unnecessary complexity for now.

We can do it in next iteration.

Reply:
Alright 👍